pcweb/pages/docs/tutorial/adding_state.py

```diff
@@ -83,18 +93,27 @@ def answer(self):
         self.chat_history.append((self.question, answer))
 """
 from pcweb.pages.docs.tutorial.frontend import qa4 as qa
+
+
 def chat1() -> rx.Component:
     return rx.box(
         rx.foreach(
-            ChatappState.chat_history,
-            lambda messages: qa(messages[0], messages[1])
+            ChatappState.chat_history, lambda messages: qa(messages[0], messages[1])
         )
     )
+
+
 def action_bar1() -> rx.Component:
     return rx.hstack(
-        rx.input(placeholder="Ask a question", on_blur=ChatappState.set_question, style=style.input_style),
+        rx.input(
+            placeholder="Ask a question",
+            on_change=ChatappState.set_question,
```

Comment:
```suggestion
            value=ChatappState.question
            on_change=ChatappState.set_question,
```

does this also need the `value` prop set, so that it can be cleared?

if so, check for other instances of input with `on_change` but no `value` as I saw at least one other